### ISSUE TITLE
Potential false negatives

### ISSUE THREAD

**Issue by mxr** [smarkets/flake8-strict#31]
The following code

```python
from __future__ import print_function


def foo1(
    arg1, arg2, arg3, arg4,
):
    print(arg1, arg2, arg3, arg4)


def foo2(arg1, arg2, arg3, arg4, ):
    print(arg1, arg2, arg3, arg4)
```

passes flake8-strict checks. I was wondering if this was expected, or if both functions should be flagged. It's not quite an S100 violation but this code seems less ideal than having each arg on its own line or all args on the same line with the function name (and no trailing comma before the paren).

Thanks!

**Comment by JaimeLennox** [comment]
I think this is expected, for now. We could probably add this as a new type of violation though.
